Stanford/Utility/Install/CreateMenuLinks.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * This class creates new menu links.
+ */
+
+namespace Stanford\Utility\Install;
+
+class CreateMenuLinks extends \AbstractInstallTask {
+
+
+  public function execute(&$args = array()) {
+    /**
+     * Create new menu links.
```

Comment:
I was wondering why you said to move it to the execute function. And why I blindly obeyed.